packages/data/addon/models/metadata/column.ts

```diff
@@ -9,22 +9,22 @@ import KegService from '../../services/keg';
 import Table from './table';
 
 export type ColumnType = 'ref' | 'formula' | 'field';
-export type NormalizedColumn = {
+export interface ColumnMetadata {
   id: string;
   name: string;
-  category: string;
-  description: string;
+  category?: string;
+  description?: string;
   tableId: string;
   source: string;
   valueType: TODO<string>;
   tags: string[];
-};
+}
 
 export type BaseExtendedAttributes = {
   description?: string;
 };
 
-export default class Column extends EmberObject {
+export default class Column extends EmberObject implements ColumnMetadata {
```

Comment:
Just to be clear, can we name the models in this PR like this:
```suggestion
export default class ColumnMetadataModel extends EmberObject implements ColumnMetadata {
```

Reply:
Oh sorry missed the model name change